samples/tab-calendar/ts/src/components/Tab.tsx

```diff
@@ -0,0 +1,380 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import React from "react";
+import "./App.css";
+import * as microsoftTeams from "@microsoft/teams-js";
+/**
```

Comment:
One line space before it.

Also, add single line comment.

Reply:
Fixed